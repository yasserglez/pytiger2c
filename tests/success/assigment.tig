let
    var output := "wrong output\n"
    var output2 := "wrong output\n"
    var i := 0
in
    output2 :=
    let
        var right := "output\n"
    in
        output := right;
        print(output);
        i := 1;
        if i then printi(i);
        print("\n");
        "output\n"
    end;
    print(output2)
end
