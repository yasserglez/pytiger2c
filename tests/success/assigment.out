output
1
output
